vulkan/vulkan_utilities.cc

```diff
@@ -10,17 +10,23 @@
 namespace vulkan {
 
 bool IsDebuggingEnabled() {
-#if OS_FUCHSIA
-  return false;
-#endif
-
 #ifndef NDEBUG
   return true;
 #else
   return false;
 #endif
 }
 
+// Whether to show Vulkan validation layer info messages in addition
+// to the error messages.
+bool ValidationLayerInfoMessagesEnabled() {
+#ifdef OS_FUCHSIA
+  return false;
+#else
+  return true;
```

Comment:
good question... i was just trying to keep non-Fuchsia platforms have the same behavior. But this might be good to have on other platforms as well— Chinmay, thoughts?